src/main/java/Events.java

```diff
@@ -0,0 +1,21 @@
+public class Events extends Task {
+    protected String eventAt;
+//    protected boolean isDone;
+
+
+    public Events(String description,String eventAt) {
+        super(description);
+        this.workType = 'E';
+        this.eventAt=eventAt;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
```

Comment:
And it gets repeated again. Imagine that this is in multiple places in your codebase. And one day, you want to change the symbol. Therefore, having all these symbols being referenced by one enum/constant will make life easier in the future.